src/database-controller/src/core/config.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
Rename this package name "core" to "common"?

Reply:
prefer the name `core`

Reply:
But it is not the core part of DB controller...

Reply:
Fine. I will change it to common